wasm-test/src/lib.rs

```diff
@@ -20,11 +24,11 @@ pub extern "C" fn check_global(encoded_allocation_of_input: u32) -> u32 {
     }
 
     hdk::debug(&hdk::APP_NAME);
-    hdk::debug(&hdk::APP_DNA_HASH);
+    hdk::debug(&hdk::APP_DNA_HASH.to_string());
     hdk::debug(&hdk::APP_AGENT_ID_STR);
-    hdk::debug(&hdk::APP_AGENT_KEY_HASH);
-    hdk::debug(&hdk::APP_AGENT_INITIAL_HASH);
-    hdk::debug(&hdk::APP_AGENT_LATEST_HASH);
+    hdk::debug(&hdk::APP_AGENT_KEY_HASH.to_string());
+    hdk::debug(&hdk::APP_AGENT_INITIAL_HASH.to_string());
+    hdk::debug(&hdk::APP_AGENT_LATEST_HASH.to_string());
```

Comment:
this is quite awful and would like to find a way to avoid having to use .to_string()

Reply:
out of curiousity (lack of knowledge), why is it awful?

Reply:
`hdk::debug()` could be generic function